src/HotChocolate/Core/test/Types.OffsetPagination.Tests/IntegrationTests.cs

```diff
@@ -656,44 +678,29 @@ protected override void Configure(IObjectTypeDescriptor<ExecutableQuery> descrip
                     .Field(t => t.FoosExecutable())
                     .Name("fooExecutable")
                     .UseOffsetPaging(
-                        options: new PagingOptions
-                        {
-                            MaxPageSize = 2,
-                            IncludeTotalCount = true
-                        });
+                        options: new PagingOptions {MaxPageSize = 2, IncludeTotalCount = true});
             }
         }
 
         public class Query
         {
             public string[] Letters => new[]
             {
-                "a",
-                "b",
-                "c",
-                "d",
-                "e",
-                "f",
-                "g",
-                "h",
-                "i",
-                "j",
-                "k",
-                "l"
+                "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l"
```

Comment:
Does this belong in one line now that its reformatted?

Reply:
I just use "chop if long or multine"